dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: page
+title: Prebid Options Supported by the Brightcove Prebid Plugin
+description: Ad Unit Reference
+top_nav_section: dev_docs
+nav_section: plugins
+pid: 10
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Prebid Options Supported by the Brightcove Prebid Plugin
+
+## Overview
+
+Configuration options are passed into the plugin in a JSON object structure.  These options can include:
+
+- Prebid options, which configure how the prebid process should be executed.
+- Rendering options, which customize the ad playback.
+
+## Plugin Prebid Options
+
+These options are used to configure how the plugin should execute the prebid process.  They are included in the same JSON structure where the rendering options are also provided.
+
+- [prebidPath](#prebidPath)
+- [biddersSpec](#biddersSpec)
+- [bidderSettings](#bidderSettings)
+- [prebidConfigOptions](#prebidConfigOptions)
+- [dfpParameters](#dfpParameters)
+- [adServerCallback](#adServerCallback)
+- [prebidTimeout](#prebidTimeout)
+- [enablePrebidCache](#enablePrebidCache)
+
+<a name="prebidPath"></a>
+### prebidPath
```

Comment:
Needs to be a space between the <a> and the ### here

Reply:
@bretg I'm looking at it on my local build and I'm not seeing where the issue is, everything looks fine on my end. What are you seeing when you do your local build?